openwisp_notifications/tests/test_notifications.py

```diff
@@ -927,3 +932,48 @@ def test_notification_cache_update(self):
         self.assertEqual(notification.target.username, 'new operator name')
         # Done for populating cache
         self.assertEqual(operator_cache.username, 'new operator name')
+
+
+class SeleniumTestNotifications(
```

Comment:
I would move this to a new dedicated test file, it's possible that over time we'll be adding more tests for the widget.